public/pages/Findings/containers/Findings/Findings.tsx

```diff
@@ -31,138 +38,59 @@ interface FindingsState {
   searchQuery: string;
   startTime: string;
   endTime: string;
+  groupBy: string;
 }
 
-const EXAMPLE_FINDINGS = [
-  {
-    id: 'finding-id-1',
-    detector_id: 'detector-id-1',
-    detector_name: 'detector1',
-    document_list: [
-      {
-        id: 'doc1',
-        index: 'index1',
-        found: true,
-        document: 'Document 1 message.',
-      },
-      {
-        id: 'doc2',
-        index: 'index1',
-        found: true,
-        document: 'Document 2 message.',
-      },
-    ],
-    index: 'index1',
-    queries: [
-      {
-        id: 'query1-id',
-        name: 'query1',
-        query: 'query source',
-        tags: ['tag1', 'tag2'],
-        category: DETECTOR_TYPES.DNS.id,
-        severity: '1',
-        description: 'A query description.',
-      },
-    ],
-    related_doc_ids: ['doc1', 'doc2'],
-    timestamp: moment.now().valueOf(),
-  },
-  {
-    id: 'finding-id-2',
-    detector_id: 'detector-id-2',
-    detector_name: 'detector2',
-    document_list: [
-      {
-        id: 'doc1',
-        index: 'index2',
-        found: true,
-        document: 'Document 1 message.',
-      },
-      {
-        id: 'doc2',
-        index: 'index2',
-        found: true,
-        document: 'Document 2 message.',
-      },
-    ],
-    index: 'index2',
-    queries: [
-      {
-        id: 'query1-id',
-        name: 'query1',
-        query: 'query source',
-        tags: ['tag1', 'tag2'],
-        category: DETECTOR_TYPES.DNS.id,
-        severity: '1',
-        description: 'A query description.',
-      },
-    ],
-    related_doc_ids: ['doc1', 'doc2'],
-    timestamp: moment.now().valueOf(),
-  },
-  {
-    id: 'finding-id-3',
-    detector_id: 'detector-id-3',
-    detector_name: 'detector3',
-    document_list: [
-      {
-        id: 'doc1',
-        index: 'index3',
-        found: true,
-        document: 'Document 1 message.',
-      },
-      {
-        id: 'doc2',
-        index: 'index3',
-        found: true,
-        document: 'Document 2 message.',
-      },
-    ],
-    index: 'index3',
-    queries: [
-      {
-        id: 'query1-id',
-        name: 'query1',
-        query: 'query source',
-        tags: ['tag1', 'tag2'],
-        category: DETECTOR_TYPES.WINDOWS.id,
-        severity: '1',
-        description: 'A query description.',
-      },
-    ],
-    related_doc_ids: ['doc1', 'doc2'],
-    timestamp: moment(moment.now()).subtract(30, 'minute').valueOf(),
-  },
+export const groupByOptions = [
+  { text: 'Log type', value: 'log_type' },
+  { text: 'Rule severity', value: 'rule_severity' },
 ];
 
 export default class Findings extends Component<FindingsProps, FindingsState> {
+  static contextType = CoreServicesContext;
+
   constructor(props: FindingsProps) {
     super(props);
     const now = moment.now();
-    const startTime = moment(now).subtract(15, 'minutes').format(DATE_MATH_FORMAT);
+    const startTime = moment(now).subtract(15, 'hours').format(DATE_MATH_FORMAT);
     this.state = {
```

Comment:
What is 15 for?  mostly for my own curiosity.

Reply:
That is a default initial value we are using for the time window being applied to search query. So, basically 15 hours means, get findings in last 15 hours